spark8t/services.py

```diff
@@ -519,6 +519,7 @@ def create(
                             "resourcename": resource_name,
                             "namespace": namespace,
                         }
+                        | extra_args
```

Comment:
Yes, I also actually like the **suggestion** and that pattern (if I have understood correctly what you mean) of using dicts over the elif statements. However, I would just leave that for another ticket thought, since it is a bit optional and off the direct scope of the PR, and (although I agree that I like the pattern) somewhat subjective to dev preference, I presume.

Reply:
I leave this for another PR, not to overload this one.